pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/filter/AllowlistFilterTest.java

```diff
@@ -22,48 +17,76 @@
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.junit.Test;
+import spark.RouteImpl;
+import spark.Service;
+import spark.route.HttpMethod;
 
 
 public class AllowlistFilterTest {
+
+    public static Service http;
     static int PORT = 6666;
     public static final String ALLOWED = "/allowed";
+    public static final String ALLOWED_TXT = "Welcome to Fantasy Island!";
     public static final String NOT_ALLOWED = "/notallowed";
     public static final String DONT_CARE = "/dontcare";
     public static final String BASEURL = "http://localhost:" + PORT;
 
     public static class TestServer {
-        static void startServer() {
-            port(6666);
+        static Service startServer() {
+            Service http;
```

Comment:
The static initialization can't be used when this test runs with other tests in our code that already spin up an instance.  To work around that case, a new instance is created.

The changes below are made to support the non-static initialization.